lib/ansible/modules/remote_management/foreman/foreman.py

```diff
@@ -8,13 +8,16 @@
 __metaclass__ = type
```

Comment:
Please rename so file starts with `_`, ie
`git mv lib/ansible/modules/remote_management/foreman/foreman.py lib/ansible/modules/remote_management/foreman/_foreman.py`

Reply:
This will clear most of the `ansible-test` issues